insights_content_template_renderer/tests/utils_test.py

```
@@ -133,32 +142,32 @@ def test_render_reports():
-        }
+        },
     }
     req = RendererRequest.parse_obj(request_data_example)
     rendered = utils.render_reports(req)
     assert RendererResponse.parse_obj(rendered).dict() == result
 
+
 def test_escape_new_line_inside_brackets():
-    input = r"{{?pydata.options == 1\n}}Option 1{{?? pydata.options == 2\n}}Option 2{{??\n}}Other option{{?}}:\n\n More text"
-    want = r"{{?pydata.options == 1}}Option 1{{?? pydata.options == 2}}Option 2{{??}}Other option{{?}}:\n\n More text"
+    input = r"{{?pydata.options == 1\n}}Option 1{{?? pydata.options == 2\n}}Option 2{{??\n}}Other option{{?}}:\n\n More text"  # noqa: E501
```

Ruff (0.15.6)

[error] Variable `input` is shadowing a Python builtin

(A001)